### ISSUE TITLE
Demonstrate positional control of steering motor using AS5601 and DRV8801

### ISSUE THREAD

**Issue by kammce** [SJSURoboticsTeam/urc-central-2021#28]
A simple program that can demonstrate a motor coupled with a magnet and a magnetic encoder to show that the output of the motor's shaft can be set to a particular angle.

- [ ] Blocked on #27
- [ ] Block on #25

**Comment by kammce** [comment]
No longer needed with the RMD-X7 motors have internal feedback and internal PID.
